docs/runners/sagemaker.md

```diff
@@ -48,26 +48,28 @@ AWS with a role that has all the required permissions to launch a SageMaker pipe
 === "Console"
     
     ```bash 
-    fondant run sagemaker <pipeline_ref> \
+    fondant run sagemaker <dataset_ref> \
+     --working-dir $S3_BUCKET \
      --role-arn $SAGEMAKER_ROLE_ARN 
     ```
     
 
 === "Python"
     
     ```python
-    from fondant.pipeline.runner import SageMakerRunner
+    from fondant.dataset.runner import SageMakerRunner
     
     runner = SageMakerRunner()
     runner.run(
-        input=<path_to_pipeline>,
+        input=<path_to_dataset>,
+        working_dir=<s3_bucket>,
         role_arn=<role_arn>,
-        pipeline_name=<pipeline_name>
+        pipeline_name=<sagemaker_pipeline_name>
     )
     ```
 
 
-Once your pipeline is running you can monitor it using the SageMaker [Studio](https://aws.amazon.com/sagemaker/studio/).
+Once your workflow is running you can monitor it using the SageMaker [Studio](https://aws.amazon.com/sagemaker/studio/).
```

Comment:
I like `workflow`. Maybe we can either use workflow or materialized dataset instead of pipeline.